agent/engine/task_manager.go

```diff
@@ -47,12 +48,13 @@ const (
 	// waitForPullCredentialsTimeout is the timeout agent trying to wait for pull
 	// credentials from acs, after the timeout it will check the credentials manager
 	// and start processing the task or start another round of waiting
-	waitForPullCredentialsTimeout         = 1 * time.Minute
-	defaultTaskSteadyStatePollInterval    = 5 * time.Minute
-	transitionPollTime                    = 5 * time.Second
-	stoppedSentWaitInterval               = 30 * time.Second
-	maxStoppedWaitTimes                   = 72 * time.Hour / stoppedSentWaitInterval
-	taskUnableToTransitionToStoppedReason = "TaskStateError: Agent could not progress task's state to stopped"
+	waitForPullCredentialsTimeout            = 1 * time.Minute
+	defaultTaskSteadyStatePollInterval       = 5 * time.Minute
+	defaultTaskSteadyStatePollIntervalJitter = 30 * time.Second
```

Comment:
By definition, jitter should vary and not fixed?

Reply:
Yes, this gets passed into the retry.AddJitter function which does the jittering. We have other constants in our code with similar naming (ie https://github.com/aws/amazon-ecs-agent/blob/61d418ea2c535c3f7356e27eec23b05a0eb90319/agent/tcs/handler/handler.go#L42)

Reply:
I see your point about naming.  I think the naming here is consistent with the rest.  It represents a rand range (`randJitter = rand.Int63n(jitter.Nanoseconds())`) so this seems like a fine name for what it is.